deploy/cr.yaml

```diff
@@ -329,6 +329,8 @@ spec:
 #        - secret:
 #            name: cluster1-pgbackrest-secrets
 #      jobs:
+#        restartPolicy: OnFailure
```

Comment:
Please do not change the defaults. We need to add the possibility of configuring it, but we need to have the old behavior by default.

Reply:
https://github.com/percona/percona-postgresql-operator/pull/969/commits/39c7804471376047415f1c05e80b50ceafcea16e